### PR TITLE
fix(ComposedModal): removed extra tooltip for close icon

### LINKED ISSUE #16724: [Bug]: hovering over close `x` in the `ComposedModal` shows 2 tooltips
### Package

@carbon/react

### Browser

Chrome, Safari, Firefox, Edge

### Package version

1.59.0

### React version

_No response_

### Description

When hovering over the close `x` in the modal we get 2 tooltips that say the same thing eg
image

relates to https://github.com/carbon-design-system/carbon/issues/16623

### Reproduction/example

https://react.carbondesignsystem.com/?path=/story/components-composedmodal--default

### Steps to reproduce

Hover over the close `x`

### Suggested Severity

Severity 4 = Unrelated to a user task, has a workaround or does not need a workaround.

### Application/PAL

IBM Cloudpak for AIOps

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

### PR DESCRIPTION
**Pull request by nandininarayanofficial**
Closes #16724

Removed additional close tooltip for ComposedModel

**Changed**

- ModalHeader.tsx
Removed title value from close icon
Updated ComposedModal-test.js

#### Testing / Reviewing

ComposedModal

### CODE REVIEW

**Comment by nandininarayanofficial** [comment]
I have read the DCO document and I hereby sign the DCO.

**Review by guidari — APPROVED** [review]
LGTM! The VO still reading as expected

**Review by tay1orjones — APPROVED** [review]

**Comment by tay1orjones** [comment]
@nandininarayanofficial this looks good to merge but needs the all contributors regenerated to resolve the merge conflict.

**Comment by nandininarayanofficial** [comment]
@tay1orjones I resolved the conflicts, please review, Thanks!

File: README.md [hunk 1]
```diff
@@ -291,7 +291,10 @@ check out our [Contributing Guide](/.github/CONTRIBUTING.md) and our
...
     <td align="center"><a href="https://github.com/ychavoya"><img src="https://avatars.githubusercontent.com/u/7907338?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Yael Chavoya</b></sub></a><br /><a href="https://github.com/carbon-design-system/carbon/commits?author=ychavoya" title="Code">💻</a></td>
-    <td align="center"><a href="https://github.com/Kilian-Collender"><img src="https://avatars.githubusercontent.com/u/37899503?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Kilian Collender</b></sub></a><br /><a href="https://github.com/carbon-design-system/carbon/commits?author=Kilian-Collender" title="Code">💻</a></td>
+    <td align="center"><img src="https://avatars.githubusercontent.com/u/37899503?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Kilian Collender</b></sub><br /><a href="https://github.com/carbon-design-system/carbon/commits?author=Kilian-Collender" title="Code">💻</a></td>
```
Comment (preetibansalui) [inline README.md]: Hey, looks like you removed the anchor tag from name and picture of this contributor.

**Review by preetibansalui — CHANGES_REQUESTED** [review]
A minor change is required.

**Comment by nandininarayanofficial** [comment]
@preetibansalui thank you for noticing this, fixed it, please review!

**Review by preetibansalui — APPROVED** [review]

**Pull request merged**